Add OpenTelemetry commitment blog post (#2440)
prometheus/docs: content/blog/2024-03-14-commitment-to-opentelemetry.md

---
title: Our commitment to OpenTelemetry
created_at: 2024-03-13
kind: article
author_name: Goutham Veeramachaneni ([@Gouthamve](https://github.com/gouthamve)) and Carrie Edwards ([@carrieedwards](https://github.com/carrieedwards))
---

*The OpenTelemetry project is an Observability framework and toolkit designed to create and manage telemetry data such as traces, metrics, and logs. It is

> gaining widespread adoption due to its consistent specification between signals and promise to reduce vendor lock-in, and it is something that we’re excited about.*

Reviewer: ```suggestion
gaining widespread adoption due to its consistent specification between signals and promise to reduce vendor lock-in which is something that we’re excited about.*
```

## Looking back at 2023

> Over the past few years, we have collaborated with the OpenTelemetry community to make sure that OpenTelemetry is well-supported in Prometheus. This led to

Reviewer: ```suggestion
Over the past few years, we have collaborated with the OpenTelemetry community to make sure that OpenTelemetry support each other Prometheus bidirectionally. This led to
```

the drafting of the official specification to convert between the two systems, as well as the implementations that allow you to ingest Prometheus metrics into
OpenTelemetry Collector and vice-versa.

Since then, we have spent a significant amount of time understanding the [challenges faced by OpenTelemetry users](https://docs.google.com/document/d/1epvoO_R7JhmHYsII-GJ6Yw99Ky91dKOqOtZGqX7Bk0g/edit?usp=sharing) when 
storing their metrics in Prometheus and based on those, explored [how we can address them](https://docs.google.com/document/d/1NGdKqcmDExynRXgC_u1CDtotz9IUdMrq2yyIq95hl70/edit?usp=sharing). Some of the changes proposed

> are extensive or involve a change to the fundamental nature of Prometheus (due to the differences in push vs pull designs). At PromCon Berlin 2023, we attempted to summarize our ideas in [one of the talks](https://www.youtube.com/watch?v=mcabOH70FqU).

Reviewer: ```suggestion
need careful considerations to avoid breaking either side's operating promises, e.g. support both push and pull. At PromCon Berlin 2023, we attempted to summarize our ideas in [one of the talks](https://www.youtube.com/watch?v=mcabOH70FqU).
```
